modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -688,6 +684,36 @@ private void _addObjectDefinitionResources(
 		}
 
 		if (objectRelationships != null) {
+			Set<String> objectRelationshipsNames = new HashSet<>();
+
+			for (ObjectRelationship objectRelationship : objectRelationships) {
+				String objectRelationshipName = objectRelationship.getName();
+
+				objectRelationshipsNames.add(objectRelationshipName);
+			}
```

Comment:
@mateussgn use the method transform here, [see](https://github.com/liferay/liferay-portal/blob/6ce0b9eb049e99a50285a324a3bd4afd76d340ab/modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-order-impl/src/main/java/com/liferay/headless/commerce/delivery/order/internal/resource/v1_0/PlacedOrderItemShipmentResourceImpl.java#L77-L84)

Reply:
Sure!
Thanks for the tip!